common/scala/src/main/scala/org/apache/openwhisk/connector/kafka/KamonMetricsReporter.scala

```diff
@@ -122,8 +122,11 @@ object KamonMetricsReporter {
   }
 
   def isCounterMetric(metric: KafkaMetric): Boolean = Try(metric.measurable()) match {
-    case Success(_: Total) => true
-    case _                 => false
+    case Success(_: CumulativeCount)                       => true
+    case Success(m) if m.getClass.getSimpleName == "Total" => true
+    case Success(m) if m.getClass.getSimpleName == "WindowedSum" && metric.metricName().name().endsWith("-total") =>
+      true
+    case _ => false
```

Comment:
it looks like `Total` was replaced with `CumulativeSum`, would capturing that in the case suffice here?

Reply:
cumulative count extends cumulative sum, i removed the two extra cases that are unnecessary addition

Reply:
if CumulativeCount extends CumulativeSum, wouldn't we want to match CumulativeSum to capture both?